test/integration/fixtures/blocks/core__comments__deprecated-1.json

```
@@ -0,0 +1,146 @@
+[
+	{
+		"name": "core/comments",
+		"isValid": true,
+		"attributes": {
+			"tagName": "div",
+			"className": "comments-post-extra has-luminous-vivid-orange-color has-text-color",
```

This is to illustrate [this problem](https://github.com/WordPress/gutenberg/pull/40506#issuecomment-1174074568).

The block-supports generated classes shouldn't be here, i.e. we'd like the following instead:

```suggestion
			"className": "comments-post-extra",
```

We'll want to change the deprecation accordingly.